sdk/iotcentral/Azure.ResourceManager.IotCentral/tests/Azure.ResourceManager.IotCentral.Tests.csproj

```diff
@@ -1,5 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <ItemGroup>
     <ProjectReference Include="..\src\Azure.ResourceManager.IotCentral.csproj" />
+    <ProjectReference Include="$(AzureCoreTestFramework)" />
```

Comment:
You don't need to include test framework here, it is already be done in [Directory.Build.Common.targets](https://github.com/Yao725/azure-sdk-for-net/blob/main/eng/Directory.Build.Common.targets#L226)